database/schema.sql

```diff
@@ -97,7 +58,48 @@ CREATE TABLE hbmember (
 	severely_mentally_impaired_ind smallint NULL,
 	period_sanction int NULL,
 	sanction_type int NULL,
-	sanction_rate_type int NULL
+	sanction_rate_type int NULL,
+	CONSTRAINT PK_hbmember PRIMARY KEY (claim_id, person_ref),
```

Comment:
Might be getting confused but person_ref isn't a PK in the Ingres table.
PK is made up of claim_id, house_id and member_id

Reply:
Yeah, I was told `person_ref` was part of the PK before I finding out that it wasn't in Ingres from our conversation.

Reply:
in which case you're perfectly correct....let's have that meeting :)

Reply:
My bad! I thought it was but it looks like Single View is wrong

Reply:
No worries!